src/utils.ts

```diff
@@ -274,14 +276,16 @@ export function filterNodesBy(nodes: any, key: 'props' | 'state', matcher: any,
     )
 }
 
-export function findReactInstance(element: any) {
+export function findReactInstance(element: HTMLElement | Node): PartialReactInstance | undefined {
+    // eslint-disable-next-line no-prototype-builtins
     if (element.hasOwnProperty('_reactRootContainer')) {
-        return element._reactRootContainer._internalRoot.current
+        return (element as HTMLElementWithReactRootContainer)._reactRootContainer._internalRoot.current as PartialReactInstance;
```

Comment:
I think with the changes suggested to `HTMLElementWithReactRootContainer ` this hoisting wouldn't be needed.